### PR TITLE
Add blog post for ORM 2.11

### PR DESCRIPTION
**Pull request by beberlei**

### CODE REVIEW

File: source/blog/2022-01-11-orm-2.11.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+    new AttributeOverride(
+        name: "id",
+        column: new Column(name: "guest_id", type: "integer", length: 140)
+    )])]
+class DDC964Admin extends DDC964User
+{
+}
+```
+
+For PHP 8.0 we have already moved Index and JoinColumn mappings to the top
```
Comment (greg0ire) [inline source/blog/2022-01-11-orm-2.11.md:144]:
```suggestion
For PHP 8.0 we have already moved `Index` and `JoinColumn` mappings to the top
```

File: source/blog/2022-01-11-orm-2.11.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+level to avoid nesting and decided not to allow nesting for these to mimic
+annotation support.
+
+## Allow arithmetic expressions within IN operator
+
+It is now possible to do arithmetic expressions or functions inside the IN operator:
```
Comment (greg0ire) [inline source/blog/2022-01-11-orm-2.11.md:150]:
```suggestion
It is now possible to use arithmetic expressions or functions inside the IN operator:
```

File: source/blog/2022-01-11-orm-2.11.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+```sql
+SELECT u FROM User u WHERE u.id IN (1 + 1, FOO(u.id))
+```
+
+## Ignore entity classes in schema tool
+
+you can now specifies Entity FQCNs to ignore during schema tool creation and comparison.
```
Comment (greg0ire) [inline source/blog/2022-01-11-orm-2.11.md:158]:
```suggestion
You can now specify Entity FQCNs to ignore during schema tool creation and comparison.
```

File: source/blog/2022-01-11-orm-2.11.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+SchemaTool will then skip these (e.g. when comparing schemas).
```
Comment (greg0ire) [inline source/blog/2022-01-11-orm-2.11.md:159]:
```suggestion
`SchemaTool` will then skip these (e.g. when comparing schemas).
```

**Review by derrabus — APPROVED** [review]

**Review by greg0ire — APPROVED** [review]

**Review by SenseException — APPROVED** [review]
Should the merge happen after the release?

**Comment by beberlei** [comment]
@SenseException yes that is the plan

**Pull request merged**